cypress/integration/swap.test.ts

```diff
@@ -37,14 +41,20 @@ context("Swap Flow", () => {
         cy.get('[data-testid="dropdownContainer"]')
           .contains(poolTokenSymbols[0])
           .click()
-        cy.react("SearchSelect").should("not.exist")
+        cy.react("SearchSelect", { options: { timeout: 1000 } }).should(
+          "not.exist",
+        )
         // cy.react("SwapTokenInput")
         //   .eq(0)
         cy.get('[data-testid="swapTokenInputFrom"]').should(
           "include.text",
           poolTokenSymbols[0],
         )
-        cy.contains("Balance:").siblings("a").should("not.have", "0.0")
```

Comment:
this was just straight up not running before - seems like a hazard of "hot.have" commands, I think `undefined.has(foo)` will always be true